Singular.js

```diff
@@ -0,0 +1,105 @@
+import {NativeEventEmitter, NativeModules} from 'react-native';
+import {version} from './package.json';
+
+const {SingularBridge} = NativeModules;
+
+const SDK_NAME = 'ReactNative';
+const SDK_VERSION = version;
+
+export class Singular {
+
+    static _singularLinkHandlerEmitter = new NativeEventEmitter(SingularBridge);
+
+    static init(singularConfig) {
+        if (!singularConfig.singularLinkHandler) {
+            SingularBridge.init(singularConfig.apikey, singularConfig.secret, singularConfig.customUserId, singularConfig.sessionTimeout);
```

Comment:
why not just pass the config object? If you add something one day you're gonna prefer it + destructure the parameters on the functions side for easier readability

Reply:
I wish... it's more complicated with RN. I can pass only their supported types